### PR TITLE
Update checkstyle config from automattic/style-config-android

### PR DESCRIPTION
**Pull request by maxme**
I fetched latest changes from `$ ./gradlew fetchStyle` and added a check to detect direct `Snackbar` usage.

Note that the `utils` subtree should be pushed after that merge.

### CODE REVIEW

**Comment by maxme** [comment]
Note, I'll open a PR on https://github.com/Automattic/style-config-android when this one is merged.

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -48,6 +48,31 @@
       <option name="CONTINUATION_INDENT_IN_PARAMETER_LISTS" value="false" />
       <option name="WRAP_EXPRESSION_BODY_FUNCTIONS" value="1" />
     </JetCodeStyleSettings>
+    <Objective-C-extensions>
```
Comment (aforcier) [inline .idea/codeStyles/Project.xml]: This one was actually up to date in WPAndroid - it should be removed from `style-config-android`. See https://github.com/wordpress-mobile/WordPress-Android/pull/9672 for background.

Comment (maxme) [reply .idea/codeStyles/Project.xml]: 👍 missed that

**Comment by aforcier** [comment]
Looks good! Verified that the `Snackbar` checkstyle rule appears to be working as intended.

I think we'll be pushing a lot of WPAndroid-specific stuff upstream but it's the best we have for now 😬

**Comment by aforcier** [comment]
:shipit:

**Pull request merged**